### PR TITLE
APP-34227: Add a variant `primary` to the `AlertPanel` component

### PR DESCRIPTION
**Pull request by alexisfontaine**
See [APP-34227](https://habxfr.atlassian.net/browse/APP-34227): ui-core: Alert panel component : add new color.

### CODE REVIEW

File: src/AlertPanel/AlertPanel.tsx [hunk 1]
```diff
@@ -11,26 +11,28 @@ import {
...
+      small,
+      success,
+      title,
+      warning,
       ...rest
     } = props
 
     return (
       <AlertBannerContainer
+        $bare={bare}
```
Comment (jean9696) [inline src/AlertPanel/AlertPanel.tsx:29]: Why did you change this ?

Comment (alexisfontaine) [reply src/AlertPanel/AlertPanel.tsx:29]: What's the point of having style-related data attributes here?

Comment (jean9696) [reply src/AlertPanel/AlertPanel.tsx:29]: To avoid the creation of class if we change the prop

Comment (alexisfontaine) [reply src/AlertPanel/AlertPanel.tsx:29]: And what's wrong about it?

Comment (jean9696) [reply src/AlertPanel/AlertPanel.tsx:29]: The previous code was not creating classes and was doing the same thing. 
Here it would be alright as we don't really change the props of `AlertPanel` but we shouldn't do this for something else like a button disabled or not. 
So the main reason is to keep the consistency

**Review by jean9696 — APPROVED** [review]
